pkgs/tools/networking/opensnitch/ui.nix

```diff
@@ -33,6 +38,7 @@ python3Packages.buildPythonApplication rec {
   preBuild = ''
     make -C ../proto ../ui/opensnitch/ui_pb2.py
     pyrcc5 -o opensnitch/resources_rc.py opensnitch/res/resources.qrc
+    sed -i 's/^import ui_pb2/from . import ui_pb2/' opensnitch/ui_pb2*
```

Comment:
Maybe adding a comment here that this is already fixed upstream and not need in versions later 1.5.1

Reply:
Oh it's been there for a while: https://github.com/evilsocket/opensnitch/blob/1.4.0/ui/Makefile

It's just that it never caused noticeable breakage until now.

I'll add a note referencing the Makefile. But, beyond that, calling the Makefile directly instead of doing it in the .nix involves more dependencies (for the i18 translations build) so I'll leave that to the permanent maintainers.